app/models/setting/general.rb

```diff
@@ -16,6 +16,7 @@ def self.default_settings
     domain = SETTINGS[:domain]
     administrator = "root@#{domain}"
     foreman_url = "#{protocol}://#{SETTINGS[:fqdn]}"
+    version = SETTINGS[:version].version
```

Comment:
In this case, I would suggest adding a setting to display the setting on the login page if someone wants to, which will be set to `false` by default.
later, in https://github.com/theforeman/foreman/blob/develop/app/helpers/login_helper.rb#L5 we can pass the setting to the login page if that setting is set to `true`